geoprocessing/src/api.config.ts

```diff
@@ -5,5 +5,5 @@
  * the relevant controllers.
  */
 export const apiGlobalPrefixes = {
-  v1: '/api/v1/geodata',
```

Comment:
are we removing the 'geodata' from the geoprocessing endpoints? I guess this is the reason why you have updated the replace from the proxy.